runtime/doc/lsp.txt

```diff
@@ -1245,30 +1245,30 @@ add_workspace_folder({workspace_folder})
 clear_references()                            *vim.lsp.buf.clear_references()*
     Removes document highlights from current buffer.
 
-code_action({options})                             *vim.lsp.buf.code_action()*
+code_action({opts})                                *vim.lsp.buf.code_action()*
```

Comment:
Maybe its something with my browser, but there seems to be some trailing whitespace here (?). Assuming you didn't modify these files manually and instead ran `make doc`, this might be a bug in the doc generator (cc @lewis6991)

Reply:
More whitespace is needed to align the tags.

Reply:
Oh duh. In that case ignore my comment lol.